provider/terraform/property_override.rb

```diff
@@ -0,0 +1,111 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'api/object'
+require 'provider/abstract_core'
+require 'provider/property_override'
+
+module Provider
+  class Terraform < Provider::AbstractCore
+    # Collection of fields allowed in the PropertyOverride section for
+    # Terraform. All fields should be `attr_reader :<property>`
+    module OverrideFields
+      attr_reader :exclude
```

Comment:
Should we move :exclude to the base?

Reply:
Added a TODO, I will do it in a separate CL to expedite this one. This will allow me to work in parallel on releasing new autogenerated resource.